src/equations/linear_scalar_advection_2d.jl

```diff
@@ -26,7 +26,7 @@ varnames(::typeof(cons2cons), ::LinearScalarAdvectionEquation2D) = SVector("scal
 varnames(::typeof(cons2prim), ::LinearScalarAdvectionEquation2D) = SVector("scalar")
 
 # Calculates translated coordinates `x` for a periodic domain
-function x_trans_periodic_2d(x, domain_length = SVector(2, 2), center = SVector(0, 0))
+function x_trans_periodic_2d(x, domain_length = SVector(10, 10), center = SVector(0, 0))
```

Comment:
I originally set the default domain_length to 2, so the entire plane is filled with gauss spheres.
For an amr indicator this isn´t very useful, because it then has to refine the entire plane.
I think in general 10 is more reasonable.